src/main/resources/docker.gradle

```diff
@@ -1,15 +1,28 @@
 apply plugin: 'com.google.cloud.tools.jib'
 
 
+ext.projectName = "${project.name}".toLowerCase()
+
+ext.getDockerImage = { ->
+    def registry = System.env.DOCKER_REGISTRY ?: ""
+    def version = System.env.DOCKER_IMAGE_VERSION ?: "latest"
+    if (registry != "" && !registry.endsWith("/")) {
+        registry = registry + "/";
+    }
+    return "${registry}${projectName}:${version}"
+}
+
 jib {
     from { image = "amazoncorretto:17" }
+    to { image = getDockerImage() }
     container {
         environment(
                 SPRING_PROFILES_ACTIVE: "default",
                 DT_DB_SCHEMA: "public",
                 DT_DB_URL: "jdbc:postgresql://localhost:5432/postgres?currentSchema=\${DT_DB_SCHEMA}",
                 DT_DB_USERNAME: "postgres",
-                DT_DB_PASSWORD: "postgres"
+                DT_DB_PASSWORD: "postgres",
+                SPRING_APPLCIATION_NAME: "${projectName}".toString()
```

Comment:
why do we need `.toString()`

Reply:
if not using toString() - this is the result 
[GStringImpl cannot be cast to java.lang.String](https://stackoverflow.com/questions/29079574/gstringimpl-cannot-be-cast-to-java-lang-string)

Reply:
hmm, then could it be
```
                SPRING_APPLCIATION_NAME: project.name.toLowerCase(),
```

Reply:
oh, there is a typo in `SPRING_APPLCIATION_NAME'